pkg/shell/hosts_dialog.jsx

```diff
@@ -475,7 +478,24 @@ class HostKey extends React.Component {
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 <p className="hostkey-type">({key_type})</p>
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
+                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">{`ssh-keyscan -t ${key_type} localhost | ssh-keygen -lf -`}</ClipboardCopy>
+                <p>{_("Optionally, paste the result below to compare fingerprint")}</p>
+                <Form>
+                    <FormGroup>
+                        <TextArea id="command-ssh-keyscan-result-paste" spellCheck="false" placeholder="# localhost:22 SSH-..."
+                                    validated={this.state.fpValidated}
+                                    onChange={(_ev, remote_fp) => { this.setState({ fpValidated: remote_fp ? remote_fp.match(fp) ? "success" : "error" : undefined }) } } />
```

Comment:
Please use some extra parentheses here to make the nesting clear: `cond1 ? (cond2 ? a : b) : c`